lib/containers/urls.pm

```diff
@@ -260,6 +261,13 @@ sub supports_image_arch {
     (grep { $_ eq $arch } @{$images_uri{$distri}{$version}{available_arch}}) ? 1 : 0;
 }
 
+# Returns a tuple of image urls.
+# If empty, no images available.
+sub get_urls_from_var {
+    my @urls = split(/,/, get_var('CONTAINER_IMAGES_TO_TEST'));
```

Comment:
I think will be nice some extra code here to avoid human mistakes, such as extra spaces,..

Reply:
Done.